extensions/wikia/ARecoveryEngine/js/recovery/tweaker.js

```diff
@@ -0,0 +1,59 @@
+/*global define*/
+define('ext.wikia.aRecoveryEngine.recovery.tweaker', [
+	'ext.wikia.adEngine.domElementTweaker',
+	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.aRecoveryEngine.recovery.helper',
+	'wikia.browserDetect',
+	'wikia.document',
+	'wikia.log'
+], function (
+	DOMElementTweaker,
+	slotTweaker,
+	recoveryHelper,
+	browser,
+	doc,
+	log
+) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.aRecoveryEngine.recovery.tweaker';
+
+	function isSupportedBrowser() {
```

Comment:
so IE and Edge are in fact not supported browsers?
(https://github.com/Wikia/app/pull/11625/files#diff-22ed2cfab091f6df9365608f88b88eb8R43)